src/Castle.Core.Tests/project.json

```diff
@@ -19,17 +22,17 @@
   },
   "compilationOptions": {
     "warningsAsErrors": true,
-    "define": [ "FEATURE_XUNITNET" ]
+    "define": [ "FEATURE_XUNITNET", "FEATURE_NETCORE_RESOURCE" ]
   },
   "resource": [
-    "Core.Tests/Resources/MoreRes/TestRes.resx"
+    "Core.Tests/Resources/MoreRes/TestRes.resx",
+    "Core.Tests/Resources/file1.txt"
   ],
   "exclude": [
-    "Components.DictionaryAdapter.Tests/**/*",
+    "AssemblyInfo.cs",
```

Comment:
Excluding AssemblyInfo.cs means the generated file doesn't get included, we need a solution to that.